src/utils/registerHandlebarTemplates.ts

```diff
@@ -80,90 +22,41 @@ export interface Templates {
 }
 
 /**
- * Read all the Handlebar templates that we need and return on wrapper object
+ * Read all default and custom Handlebar templates that we need and return on wrapper object
  * so we can easily access the templates in out generator / write functions.
  */
-export function registerHandlebarTemplates(): Templates {
+export function registerHandlebarTemplates(customTemplates?: string): Templates {
+    const templatesDefault = path.resolve(__dirname, 'templates');
+    const templatesRuntime = path.resolve(__dirname, 'runtime');
```

Comment:
You cannot run this process then in parallel multiple. This will fail CI (even for us, we can generate v1 and v2 at once) for a lot of people I guess. Use process.pid to create temp directory.